<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageHistory</class>
 <widget class="QDialog" name="MessageHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messages</string>
  </property>
  <property name="windowIcon">
   <iconset resource="taki.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="lbMsgNotFound">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No messages found.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QFrame" name="frMsgView">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolButton" name="btMsgBack">
          <property name="text">
           <string>Back</string>
          </property>
          <property name="icon">
           <iconset resource="taki.qrc">
            <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="btMsgNew">
          <property name="text">
           <string>Compose</string>
          </property>
          <property name="icon">
           <iconset resource="taki.qrc">
            <normaloff>:/icons/edit.png</normaloff>
            <disabledoff>:/icons/edit-disbl.png</disabledoff>:/icons/edit.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QToolButton" name="btMsgDelete">
          <property name="text">
           <string>Delete all</string>
          </property>
          <property name="icon">
           <iconset resource="taki.qrc">
            <normaloff>:/icons/delete-all.png</normaloff>
            <disabledoff>:/icons/delete-all-disbl.png</disabledoff>:/icons/delete-all.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTreeView" name="twMsgHistory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QFrame" name="frBBControls">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="bbFrameLabel">
          <property name="text">
           <string>Messages</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="taki.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btMsgBack</sender>
   <signal>clicked()</signal>
   <receiver>MessageHistory</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twMsgHistory</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MessageHistory</receiver>
   <slot>onChatSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChatSelected(QTreeWidgetItem*,int)</slot>
  <slot>onChatSelected(QModelIndex)</slot>
 </slots>
</ui>
